Tone/component/filter/LowpassCombFilter.ts

```diff
@@ -0,0 +1,91 @@
+import { Param } from "../../core/context/Param";
+import { InputNode, OutputNode, ToneAudioNode, ToneAudioNodeOptions } from "../../core/context/ToneAudioNode";
+import { Frequency, NormalRange, Time } from "../../core/type/Units";
+import { optionsFromArguments } from "../../core/util/Defaults";
+import { RecursivePartial } from "../../core/util/Interface";
+import { Signal } from "../../signal/Signal";
+import { FeedbackCombFilter } from "./FeedbackCombFilter";
+import { Filter } from "./Filter";
+
+interface LowpassCombFilterOptions extends ToneAudioNodeOptions {
+	delayTime: Time;
+	resonance: NormalRange;
+	dampening: Frequency;
+}
+
+/**
+ *  Tone.Lowpass is a lowpass feedback comb filter. It is similar to
+ *  Tone.FeedbackCombFilter, but includes a lowpass filter.
+ */
+export class LowpassCombFilter extends ToneAudioNode<ToneAudioNodeOptions> {
```

Comment:
Similar to the previous comment, the end of this line should be `extends ToneAudioNode<LowpassCombFilterOptions>`